changelogs/fragments/config_updates.yml

```diff
@@ -0,0 +1,3 @@
+minor_changes:
+  - ansible-config can now list and dump for specific documentable plugins by specifying them in the command line
+  - ansible-config has new 'init' option to create, highly commented, example configurations as ini, env vars or yaml variable files
```

Comment:
The comments in the code say that YAML isn't supported yet.

Reply:
yaml is 'vars in yaml file' .. not the yaml key .. but will reword